modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/APIBuilderUI.macro

```diff
@@ -233,6 +256,20 @@ definition {
 			value1 = "\RETURN");
 	}
 
+	macro selectEndPointScope {
```

Comment:
If we don't run into an error when running our test, then by default, we don't add waitFor...  :smile: